### PR TITLE
Chris mon 02 06 17

### PR DESCRIPTION
**Pull request by cjl8zf**

### CODE REVIEW

**Comment by cjl8zf** [comment]
Hey @MareoRaft I added you as a reviewer for this week

**Comment by MareoRaft** [comment]
@cjl8zf For mathematical changes, do you prefer for me to make comments, or make the changes directly to the file?

File: Algebraic Topology/algebraic-topology.tex [hunk 1]
```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
...
+
+  $\R P^n = \{ \R \dot (1, a_1,,, a_{n-1})\} \isom \R^{n}$.
+  $\bunion \{ \R \dot (0, a_1,,, a_{n-1})\} \isom \R P^{n-1}$.
+\end{example}
+
+\mydate{d7}{6}{2}{2017}
+
+We wish to show how \(\R P^n\) is a \(CW\)-complex. First let the
+\(n-1\) skeleton be \(\R P^{n-1}\). We add the \(n\) cell \(D^n\) to
+structure under the gluing map \(S^{n} \to \R P^{n-1}\) where
```
Comment (MareoRaft) [inline Algebraic Topology/algebraic-topology.tex]: Maybe he meant S^{n-1}?  The picture that I do understand in my mind is the representation of RP^2 (lines in R3) as a CW complex.  It looks like a disk D2 with half the circumference identified with the other half of the circumference.

In that particular case, what would the glueing map be?  Would it be glueing the two circumference halves (S^1) to itself?

File: Algebraic Topology/algebraic-topology.tex [hunk 2]
```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
...
+  constructed as follows for each \(n \in \Z\) consider the circle of
+  radius \(1/n\) tangent to the \(y\)-axis. All of these circles
+  together with the subspace topology yield a topological space, call
+  it \(X\).
+
+  The only hope for putting a \(CW\) structure on this pace would be
+  to consider one \(0\)-cell along with a countable amount of
+  \(1\)-cells. However recall that the \(CW\)-structure gain their
+  topology through the quotient topology, that is, \(\mathcal{O}\) is
+  open in \(X\) if and only if \(q^{-1}(X)\) is open, where \(q\) is
```
Comment (MareoRaft) [inline Algebraic Topology/algebraic-topology.tex]: I think he meant q^{-1}(\mathcal{O}).

File: Algebraic Topology/algebraic-topology.tex [hunk 3]
```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
...
+  The topological space known as the \emph{Hawaiian earring} is
+  constructed as follows for each \(n \in \Z\) consider the circle of
+  radius \(1/n\) tangent to the \(y\)-axis. All of these circles
+  together with the subspace topology yield a topological space, call
+  it \(X\).
+
+  The only hope for putting a \(CW\) structure on this pace would be
+  to consider one \(0\)-cell along with a countable amount of
+  \(1\)-cells. However recall that the \(CW\)-structure gain their
+  topology through the quotient topology, that is, \(\mathcal{O}\) is
```
Comment (MareoRaft) [inline Algebraic Topology/algebraic-topology.tex]: They do? Can you give an example?  What is the quotient map in this case?

File: Algebraic Topology/algebraic-topology.tex [hunk 4]
```diff
@@ -431,17 +436,202 @@ \section{Functoriality of $\fund$}
...
+Next we show how to compute \(\pi_1\) of a few
+\(CW\)-structures. Using Morse Theory one may show that any compact
+manifold admits a \(CW\)-structure.
+
+To compute \(\pi_1\) of the torus one can first decompose it as a
+\(CW\)-complex as follows
+\begin{align*}
+  X_0 &= *\\
+  X_1 &= S^1 \vee S^1\\
+  X_2 &= D_2
```
Comment (MareoRaft) [inline Algebraic Topology/algebraic-topology.tex:714]: Maybe he meant D^2 ?  I still don't get CW-complexes.  For the torus, X_0 and X_1 make sense, but this X_2 does not make sense to me.  Do we eventually get the torus itself (S^1 \x S^1) for one of the X_n's ?

Comment (MareoRaft) [reply Algebraic Topology/algebraic-topology.tex:714]: Or maybe he meant T^2

**Review by MareoRaft — CHANGES_REQUESTED** [review]
This is good Chris.  I feel like I'm *this* close to understanding CW-complexes!  Or should I say, CW-complices?

There were some comments I made that weren't included as part of "the review".  But you should still be able to see them under the "files changed" tab.

**Comment by MareoRaft** [comment]
Or rather, the comments are hidden in the "Show outdated" links on this page.

File: Algebraic Topology/algebraic-topology.tex (hunk 4 above)
Comment (cjl8zf) [reply Algebraic Topology/algebraic-topology.tex:714]: what I was trying to say was that you glue on a single 2-cell, not sure the correct notation, any suggestions?

Comment (MareoRaft) [reply Algebraic Topology/algebraic-topology.tex:714]: I understand CW complexes better now.  So the result of gluing on a single 2-cell is that you do in fact receive the torus as X2.  So X_2 &= M_1, the torus.

**Comment by MareoRaft** [comment]
I merged this but you can still comment / review if you want.

**Pull request merged**